provisioner.go

```diff
@@ -369,42 +380,40 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 			},
 		},
 	}
-	lpvVolumeMounts := []v1.VolumeMount{
-		{
-			Name:      "data",
-			ReadOnly:  false,
-			MountPath: parentDir,
-		},
-		{
-			Name:      "script",
-			ReadOnly:  false,
-			MountPath: "/script",
-		},
-	}
 	lpvTolerations := []v1.Toleration{
 		{
 			Operator: v1.TolerationOpExists,
 		},
 	}
 	helperPod := p.helperPod.DeepCopy()
 
+	scriptMount := addVolumeMount(&helperPod.Spec.Containers[0].VolumeMounts, helperScriptVolName, helperScriptDir)
+	scriptMount.MountPath = helperScriptDir
```

Comment:
Why is scriptMount.MountPath assigned again?

Reply:
`addVolumeMount` would not overwrite the `MountPath` that the Pod template specifies (there are potential use cases where the volume path within the container must equal the path on the host and therefore users would want to change the path as well within the Pod template).

By explicitly assigning `MountPath` of the script volume mount I ensured that users cannot overwrite the `/script` path within the Pod template in the same way since the `cmd`` that the calling code provides relies on it (I didn't see a reason why users would want to configure a script path other than the hardcoded one).

Reply:
Make sense. Thank you.